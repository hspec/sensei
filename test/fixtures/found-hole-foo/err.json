{"version":"1.0","ghcVersion":"ghc-9.10.0","span":{"file":"A","start":{"line":2,"column":5},"end":{"line":2,"column":6}},"severity":"Error","code":88464,"message":["Found hole:\n  _ :: String\n       -> String\n       -> String\n       -> String\n       -> String\n       -> String\n       -> String\n       -> String","In an equation for `a': a = _","Relevant bindings include\n  a :: String\n       -> String\n       -> String\n       -> String\n       -> String\n       -> String\n       -> String\n       -> String\n    (bound at A:2:1)\nValid hole fits include\n  a :: String\n       -> String\n       -> String\n       -> String\n       -> String\n       -> String\n       -> String\n       -> String\n    (bound at A:2:1)\n  mempty :: forall a. Monoid a => a\n    with mempty @(String\n                  -> String\n                  -> String\n                  -> String\n                  -> String\n                  -> String\n                  -> String\n                  -> String)\n    (imported from `Prelude' at A:1:1\n     (and originally defined in `ghc-internal-9.1001.0:GHC.Internal.Base'))"],"hints":[]}
