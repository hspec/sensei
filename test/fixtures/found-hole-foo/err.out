A:2:5: error: [GHC-88464]
    • Found hole:
        _ :: String
             -> String
             -> String
             -> String
             -> String
             -> String
             -> String
             -> String
    • In an equation for `a': a = _
    • Relevant bindings include
        a :: String
             -> String
             -> String
             -> String
             -> String
             -> String
             -> String
             -> String
          (bound at A:2:1)
      Valid hole fits include
        a :: String
             -> String
             -> String
             -> String
             -> String
             -> String
             -> String
             -> String
          (bound at A:2:1)
        mempty :: forall a. Monoid a => a
          with mempty @(String
                        -> String
                        -> String
                        -> String
                        -> String
                        -> String
                        -> String
                        -> String)
          (imported from `Prelude' at A:1:1
           (and originally defined in `ghc-internal-9.1001.0:GHC.Internal.Base'))

